package system

func FailWithError(errMsg string) {
	// WIP: ちゃんとしたエラーにする
	panic(errMsg)
}
